tests/features/content-language-switcher.feature

```diff
@@ -9,27 +9,41 @@ Feature: Content language selector
     Given the following "Demo translatable page" content item:
       | Title | Page title |
       | Body  | Page body  |
-    And the following "Italian" translation for the "Demo translatable page" with title "Page title":
-      | Title | Titolo pagina |
-      | Body  | Testo pagina  |
     And the following "Greek" translation for the "Demo translatable page" with title "Page title":
       | Title | Τίτλος σελίδας  |
       | Body  | Σελίδα κειμένου |
     And the following "Spanish" translation for the "Demo translatable page" with title "Page title":
       | Title | Título de página  |
       | Body  | Texto de página   |
+    And the following "Italian" translation for the "Demo translatable page" with title "Page title":
+      | Title | Titolo pagina |
+      | Body  | Testo pagina  |
 
   Scenario: Visitor navigating to an available translation shouldn't see the language selector
-    Given I visit the "Page title" content
+    When I visit the "Page title" content
     Then I should see the heading "Page title"
     And I should see "Page body"
-    And I should not see the link "Spanish" in the "page header" region
+
+    And I should not see the link "ελληνικά" in the "page content" region
+    And I should not see the link "español" in the "page content" region
+    And I should not see the link "italiano" in the "page content" region
+
+  Scenario: Visitor navigating to an available translation shouldn't see the language selector
+    When I visit the "Page title" content
+    And I click "italiano"
+    Then I should see the heading "Titolo pagina"
+    And I should see "Testo pagina"
+
+    And I should not see the link "ελληνικά" in the "page content" region
+    And I should not see the link "English" in the "page content" region
+    And I should not see the link "español" in the "page content" region
```

Comment:
We can remove the trailing ` region` from all these steps, it will make them more readable, like:
```gherkin
    And I should not see the link "ελληνικά" in the "page content"
    And I should not see the link "English" in the "page content"
    And I should not see the link "español" in the "page content"
```